core/lib/Thelia/Command/Skeleton/Module/composer.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "thelia/%%COMPOSERNAME%%-module",
```

Comment:
I don't know if it's a good idea to pre-fill the name with thelia vendor.

something like this could be better : 

```
name: "your-vendor/%%COMPOSERNAME%%-module"
```